api/src/main/java/io/opentelemetry/trace/TraceFlags.java

```diff
@@ -63,191 +50,14 @@ public static int getSize() {
-   * @param destOffset the starting offset in the destination buffer.
-   * @throws IndexOutOfBoundsException if {@code destOffset + 2} is greater than {@code
-   *     dest.length}.
-   * @since 0.1.0
-   */
-  public void copyLowerBase16To(char[] dest, int destOffset) {
-    BigendianEncoding.byteToBase16String(options, dest, destOffset);
-  }
-
-  /**
-   * Returns the lowercase base16 encoding of this {@code TraceFlags}.
-   *
-   * @return the lowercase base16 encoding of this {@code TraceFlags}.
-   * @since 0.1.0
-   */
-  public String toLowerBase16() {
-    char[] chars = new char[BASE16_SIZE];
-    copyLowerBase16To(chars, 0);
-    return new String(chars);
-  }
-
-  /**
-   * Returns a new {@link Builder} with default options.
-   *
-   * @return a new {@code Builder} with default options.
-   * @since 0.1.0
-   */
-  public static Builder builder() {
-    return new Builder(DEFAULT_OPTIONS);
-  }
-
-  /**
-   * Returns a new {@link Builder} with all given options set.
-   *
-   * @param traceFlags the given options set.
-   * @return a new {@code Builder} with all given options set.
-   * @since 0.1.0
-   */
-  public static Builder builder(TraceFlags traceFlags) {
-    return new Builder(traceFlags.options);
-  }
-
-  /**
-   * Returns a boolean indicating whether this {@code Span} is part of a sampled trace and data
-   * should be exported to a persistent store.
-   *
-   * @return a boolean indicating whether the trace is sampled.
-   * @since 0.1.0
-   */
-  public boolean isSampled() {
-    return hasOption(IS_SAMPLED);
-  }
-
-  @Override
-  public boolean equals(@Nullable Object obj) {
-    if (obj == this) {
-      return true;
-    }
-
-    if (!(obj instanceof TraceFlags)) {
-      return false;
-    }
-
-    TraceFlags that = (TraceFlags) obj;
-    return options == that.options;
-  }
-
-  @Override
-  public int hashCode() {
-    return Arrays.hashCode(new byte[] {options});
-  }
-
-  @Override
-  public String toString() {
-    return "TraceFlags{sampled=" + isSampled() + "}";
-  }
-
-  /**
-   * Builder class for {@link TraceFlags}.
-   *
-   * @since 0.1.0
-   */
-  public static final class Builder {
-    private byte options;
-
-    private Builder(byte options) {
-      this.options = options;
-    }
-
-    /**
-     * Sets the sampling bit in the options.
-     *
-     * @param isSampled the sampling bit.
-     * @return this.
-     * @since 0.1.0
-     */
-    public Builder setIsSampled(boolean isSampled) {
-      if (isSampled) {
-        options = (byte) (options | IS_SAMPLED);
-      } else {
-        options = (byte) (options & ~IS_SAMPLED);
-      }
-      return this;
-    }
-
-    /**
-     * Builds and returns a {@code TraceFlags} with the desired options.
-     *
-     * @return a {@code TraceFlags} with the desired options.
-     * @since 0.1.0
-     */
-    public TraceFlags build() {
-      return fromByte(options);
-    }
+  public static boolean getDefault() {
```

Comment:
Just for context, this is a big reason for removing / moving `TraceFlags` to w3c - it's pretty weird for the default `TraceFlags` to just be a boolean instead of `00` :)

Reply:
Yep. I agree. I'll create an issue for it, as a follow-up.

Reply:
#1632